tidy3d/components/simulation.py

```diff
@@ -3245,6 +3249,24 @@ def _post_init_validators(self) -> None:
         self._validate_tfsf_nonuniform_grid()
         self._validate_nonlinear_specs()
         self._validate_custom_source_time()
+        self._validate_mode_object_bends()
+
+    def _validate_mode_object_bends(self) -> None:
+        """Error if any mode sources or monitors with bends have a radius that is too small."""
+        for imnt, monitor in enumerate(self.monitors):
+            if isinstance(monitor, AbstractModeMonitor):
+                validate_mode_plane_radius(
+                    mode_spec=monitor.mode_spec,
+                    plane=monitor.geometry,
+                    msg_prefix=f"Monitors[{imnt}] ",
```

Comment:
do we want to put this in single quotes `'` and also use lowercase like `monitors[...]`?

Reply:
done